File: Daily Challenges/2021/May/Day 3/Running-the-Sum-of-1D-Array.java
class Solution {
    public int[] runningSum(int[] nums) {
        int[] s = new int[nums.length];
        s[0] = nums[0];
        for(int i = 1; i<nums.length; i++){
            s[i] = s[i-1]+nums[i];
        }
        return s;
    }
}

File: Daily Challenges/2021/May/Day 3/Running-the-Sum-of-1D-Array.cpp
class Solution {
public:
    vector<int> runningSum(vector<int>& nums) {
        vector<int>s;
        s.push_back(nums[0]);
        for(int i = 1; i<nums.size(); i++){
            s.push_back(s[s.size()-1]+nums[i]);
        }
        return s;
    }
};
